app/routes/announcements/components/AnnouncementsList.tsx

```diff
@@ -1,56 +1,37 @@
 import { Flex } from '@webkom/lego-bricks';
+import { useEffect } from 'react';
+import { fetchAll } from 'app/actions/AnnouncementsActions';
 import { Content, ContentMain } from 'app/components/Content';
-import type { ActionGrant } from 'app/models';
-import type { ID } from 'app/store/models';
-import type {
-  DetailedAnnouncement,
-  ListAnnouncement,
-} from 'app/store/models/Announcement';
+import { selectAnnouncements } from 'app/reducers/announcements';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
 import AnnouncementItem from './AnnouncementItem';
 import AnnouncementsCreate from './AnnouncementsCreate';
 import styles from './AnnouncementsList.css';
 
-type Props = {
-  announcements: Array<ListAnnouncement>;
-  actionGrant: ActionGrant;
-  sendAnnouncement: (id: ID) => Promise<unknown>;
-  createAnnouncement: (
-    announcement: DetailedAnnouncement & { send: boolean }
-  ) => Promise<unknown>;
-  deleteAnnouncement: (id: ID) => Promise<unknown>;
-  handleSubmit: (arg0: (...args: Array<any>) => any) => void;
-  invalid: string;
-  pristine: string;
-  submitting: string;
-};
+const AnnouncementsList = () => {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
```

Comment:
`usePreparedEffect`? I know SSR doesn't work yet, but it doesn't hurt to add it in now.

Reply:
I'm aware heh